boot.php

```diff
@@ -15,7 +15,7 @@
 use GameBoy\Keyboard;
 
 if (PHP_VERSION_ID >= 70000) {
-    set_exception_handler(function (Error $exception) {
+    set_exception_handler(function (Throwable $exception) {
```

Comment:
Because bare 'Throwable' does not cause an error or warning.
'\Throwablw' also does not cause an error or warning.
I don't know why.
Is it preferable to add \ ?